server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1462,6 +1462,292 @@ public void shouldCreateProfilesFromMultipleAgentConfigsAcrossPipelines_asPartOf
         assertThat(profiles.get(1), is(expectedAWSProfile));
     }
 
+    @Test
+    public void shouldCreateAViewElementInEachTemplateIfPipelineGroupAdminsExist_asPartOfMigration89() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
```

Comment:
Correction: 
> only pipeline group admins who had pipelines depending on templates could see those templates. Right?

No. When a pipeline group admin wants to create a pipeline using a template, they'll need to view all templates. So, all pipeline group admins ended up seeing the configuration of all templates.